### PR TITLE
Add imi60 support

### PR DESCRIPTION
**Pull request by key10iq**
This PR adds Vial and QMK support for imi60. It is a PCB for two future keyboards currently in prototyping.
Flashed and tested successfully.
PCB hasn't been added to the QMK repository.

### CODE REVIEW

File: keyboards/keyten/imi60/config.h
Comment (lesshonor) [inline keyboards/keyten/imi60/config.h:1]: Delete this file; it is unnecessary.

File: keyboards/keyten/imi60/imi60.c
Comment (lesshonor) [inline keyboards/keyten/imi60/imi60.c:1]: Delete this file; it is unnecessary.

File: keyboards/keyten/imi60/imi60.h
Comment (lesshonor) [inline keyboards/keyten/imi60/imi60.h:1]: Delete this file; it is unnecessary (the matrix information is already in `info.json`)

File: keyboards/keyten/imi60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	),
+
+	[3] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	),
+
```
Comment (lesshonor) [inline keyboards/keyten/imi60/keymaps/default/keymap.c]: Don't need to have extra blank layers
```suggestion
)
```

File: keyboards/keyten/imi60/keymaps/vial/keymap.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	),
+
+	[3] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+	),
+
```
Comment (lesshonor) [inline keyboards/keyten/imi60/keymaps/vial/keymap.c]: Don't need to have extra blank layers, these will be automatically filled in
```suggestion
)
```

File: keyboards/keyten/imi60/keymaps/vial/rules.mk [hunk 3]
```diff
@@ -0,0 +1,8 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIAL_INSECURE = yes
```
Comment (lesshonor) [inline keyboards/keyten/imi60/keymaps/vial/rules.mk]:
```suggestion
```
[Per the documentation](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination):
> keyboards which enable `VIAL_INSECURE` will not be accepted to the main `vial-qmk` repository.

File: keyboards/keyten/imi60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
+
+LTO_ENABLE = yes
```
Comment (lesshonor) [inline keyboards/keyten/imi60/rules.mk]: You already set all this information in `info.json` and that is where it belongs. The `BOOTLOADER` here actually contradicts what you've got there, so make sure it is correct in `info.json`...

```suggestion
# This file intentionally left blank
```

File: keyboards/keyten/imi60/info.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
+{
+    "manufacturer": "keyten",
+    "keyboard_name": "imi60",
+    "maintainer": "qmk",
```
Comment (lesshonor) [inline keyboards/keyten/imi60/info.json]:
```suggestion
    "maintainer": "key10iq",
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Review by lesshonor — CHANGES_REQUESTED** [review]
[Doesn't build](https://github.com/lesshonor/vial-qmk/actions/runs/5179271237/jobs/9331934518). Here's [a hint](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination).

Don't forget to test these changes on your PCB.

**Comment by key10iq** [comment]
> [Doesn't build](https://github.com/lesshonor/vial-qmk/actions/runs/5179271237/jobs/9331934518). Here's [a hint](https://get.vial.today/docs/porting-to-vial.html#6-set-up-a-secure-unlock-combination).
> 
> Don't forget to test these changes on your PCB.

Tested. Works.
Thanks for the tip!

**Review by lesshonor — APPROVED** [review]

**Pull request merged**